book/08_Flood_analysis/3_Retrieving_FloodData.md

```diff
@@ -148,23 +141,32 @@ granules
 ```
 
 ```python
-def filter_by_values(df, bad_values=[253, 255], filter_percentage=10):
+type(results[0])
+
+```
+
+```python
+def filter_search_by_cc(results, cloud_threshold=10):
```

Comment:
We're going to build this function in a notebook & explain its function carefully. I think this may belong in a notebook in which we explain the [PySTAC](https://pystac.readthedocs.io/en/stable/) API.